x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -38,6 +54,13 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
         }}
       />
     </section>
+  ) : (
+    <EuiEmptyPrompt
+      data-test-subj="alerts-table-no-configuration"
+      iconType="logoSecurity"
```

Comment:
The table can be rendered in different solutions (plugins) so I think we should use a generic logo for all. For example, the `watchesApp` that is [being used](https://github.com/elastic/kibana/blob/e8f59a588886f10eb4d37bfd80742a9239190ee9/x-pack/plugins/triggers_actions_ui/public/plugin.ts#L131) by the plugin

Reply:
Good catch! I didn't pay much attention to the logo :)